core/fuzzer.cpp

```diff
@@ -40,6 +27,18 @@ p4::v1::FieldMatch_Ternary P4RuntimeFuzzer::produceFieldMatch_Ternary(int bitwid
     return protoTernary;
 }
 
+p4::v1::FieldMatch_Range P4RuntimeFuzzer::produceFieldMatch_Range(int bitwidth) {
+    p4::v1::FieldMatch_Range protoRange;
+    auto value1 = produceBytes(bitwidth);
```

Comment:
We could add a helper that generates a random value within a range. That could help here because then we can set the first generated value as max.